src/subapps/admin/components/Settings/ViewsSubView.tsx

```diff
@@ -236,7 +228,56 @@ const ViewsSubView = () => {
     }
   );
 
-  const columns: ColumnsType<TViewType> = [
+  const fetchIndexingErrorsOnDemand = async ({
+    nexus,
+    apiEndpoint,
+    orgLabel,
+    projectLabel,
+    viewId,
+  }: {
+    nexus: NexusClient;
+    apiEndpoint: string;
+    orgLabel: string;
+    projectLabel: string;
+    viewId: string;
+  }) => {
+    try {
+      // Fetch indexing errors for the specified view
+      const indexingErrors = await fetchIndexingErrors({
+        nexus,
+        apiEndpoint,
+        orgLabel,
+        projectLabel,
+        viewId,
+      });
+
+      return indexingErrors;
+    } catch (error) {
+      console.error('Error fetching indexing errors on demand', error);
+      notification.error({
+        message: `Error fetching indexing errors for the selected view`,
+        description: '',
+      });
+
+      Sentry.captureException(error, {
+        extra: {
+          orgLabel,
+          projectLabel,
+          viewId,
+          error,
+        },
+      });
+
+      return {
```

Comment:
I don't think that returning the object when Error is a good idea, catching an error must not return a result object unless there is a good reason for that.
returning `null` is better